Swiftfin tvOS/Views/VideoPlayer/Overlays/Overlay.swift

```diff
@@ -26,6 +26,8 @@ extension VideoPlayer {
         private var confirmCloseWorkItem: DispatchWorkItem?
         @State
         private var currentOverlayType: VideoPlayer.OverlayType = .main
+        @State
+        private var isPlaying = true
```

Comment:
```suggestion
@EnvironmentObject
private var videoPlayerManager: VideoPlayerManager
```

Reply:
Probably best to place this `@EnvironmentObject` above with the other environment objects, its just that I couldn't find a way to use GitHub's suggestion feature on lines that have not been changed already.

Reply:
Yes, ti should go with the other `EnvironmentObject`s.